README.md

```diff
@@ -1,27 +1,25 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg" width="512" height="256">
+  <br>
+  <a href="https://www.rust-lang.org/">Website</a> |
+  <a href="https://www.rust-lang.org/learn/get-started">Getting started</a> |
+  <a href="https://www.rust-lang.org/learn">Learn</a> |
+  <a href="https://www.rust-lang.org/learn#learn-use">Documentation</a> |
+  <a href="CONTRIBUTING.md">Contributing</a>
+</p>
 
 This is the main source code repository for [Rust]. It contains the compiler,
 standard library, and documentation.
 
 [Rust]: https://www.rust-lang.org/
 
-**Note: this README is for _users_ rather than _contributors_.**
-If you wish to _contribute_ to the compiler, you should read
-[CONTRIBUTING.md](CONTRIBUTING.md) instead.
+## Why Rust?
 
-<details>
-<summary>Table of Contents</summary>
+- **Performance:** Fast and memory-efficient, suitable for critical services, embedded devices, and easily integrate with other languages.
 
-- [Quick Start](#quick-start)
-- [Installing from Source](#installing-from-source)
-- [Getting Help](#getting-help)
-- [Contributing](#contributing)
-- [License](#license)
-- [Trademark](#trademark)
+- **Reliability:** Our rich type system and ownership model ensure memory and thread safety, reducing bugs at compile-time.
 
-</details>
+- **Productivity:** Comprehensive documentation, user-friendly compiler, and advanced tooling including package manager, build tool, auto-formatter, and editor support.
```

Comment:
This deviation from the website's way of expressing it is again driven by "this is the README for the compiler's source repo, so we should use it to introduce people to what they can find in this repo or related repos".

Reply:
> I feel like our compiler is slightly more-precise than "user-friendly": we want good diagnostics, and we treat bad diagnostics as bugs if they are at all improvable.

Completely agree, but I don't know how to say that with only 2/3 words. I therefor changed it to "user-focused" instead of "user-friendly" to emphasise the fact that we care about the users.

> I also think we can go ahead and cross-link here: we can identify that we ship this stuff via Rustup, and we can name cargo, rustfmt, rust-analyzer, and what they are, with hyperlinks.

Also agree, I've added a hyperlink to all the major tools we ship.